.travis.yml

```diff
@@ -3,9 +3,7 @@ language: php
 sudo: false
 
 php:
-  - 7.1
-  - 7.2
-  - 7.3
+  - '7.4snapshot'
```

Comment:
Can you update the pull request just adding the `7.4snapshot`?

Reply:
Yes, if you solve the above exception.

Reply:
@kylekatarnls Do you feel like investigating this one?

Reply:
OK, I will take a look. It may be not harder than a `if (!is_int($foo))` but I will check to be sure.

Reply:
I will base it on my branch for carbon-macro to see if there is no PHP 7.4 incompatibility in my changes neither.

Reply:
Hum, a bit harder than I thought, I think it's something to fix in Laravel:

Laravel do:
`array_unshift($parameters, $command);` putting `$command` (code:analyse) at the top of the parameters with a `0` index (don't see why) while Symfony expect only string keys for `$parameters`

Reply:
Error seems to have been fixed in Symfony:
https://github.com/symfony/symfony/pull/32806/files

Reply:
Very well done, thanks for the investigation!!